database/apid/src/main/java/com/example/javaopencommerce/warehouse/WarehouseItemDetails.java

```diff
@@ -0,0 +1,24 @@
+package com.example.javaopencommerce.warehouse;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "warehouse_item_details")
+class WarehouseItemDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String shortDescription;
+    private String location;
+    private double unitWeight; // what bout unit of measure?
+    private double unitDimension; // what bout unit of measure?
```

Comment:
It would be good to have a suffix 'Entity' for all entities. 
The unit may be another field, an enum and some string in the database.

If this is an information for a storing the product, it should have the 3 dimensions: height, width, depth.

Reply:
About dimenstions etc - i will those are just placeholders for now, to somehow imagine direction of changes :d 

About enttiy - good point, missed that!